save-common-test/src/commonNonJsMain/kotlin/org/cqfn/save/Utils.kt

```diff
@@ -0,0 +1,34 @@
+/**
+ * Common utilities for testing
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package org.cqfn.save
+
+import okio.FileSystem
+import okio.Path
+import kotlin.random.Random
+
+/**
+ * Marks method to be executed before all tests, supported on JVM and Native
+ */
+expect annotation class BeforeClass
```

Comment:
I don't see the usage of these annotations in this PR, is it for future enhancements?

Reply:
Yeah, I initially wanted to use these annotations, but then implemented resources cleanup as a gradle task. Decided to keep annotations, because they probably will be useful in the future.